Will Heikoff Documentation

```diff
@@ -235,5 +235,13 @@ Researching to figure out why only 151,188 blocks are available in the data set,
 Wrote program in R called: ZIPs_Tot_Emp.r located in G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File,
 	in order to view and format zbp00detail.txt more efficiently
 	Imported the “sqldf” package into R
-Formatted data by excluding all data of which didn’t have “31----“(manufacturing employment) in its naics variable, resulting in the 
-	26586 zip codes with manufacturing data
+Formatted by excluding all data of which didn’t have “31----“( total manufacturing employment, including all naics codes starting with 
+	31, 32, and 33; as seen in https://www2.census.gov/programs-surveys/cbp/technical-documentation/reference/naics-descriptions/
```

Comment:
I'd like to see a verbal description that confirms what we believed to be true from looking at this listing.